### PR TITLE
Added videos to "Customize DevTools"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/index?branch=pr-en-us-2161

Before:
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/
Related:
https://docs.microsoft.com/en-us/microsoft-edge/dev-videos/

### CODE REVIEW

**Comment by captainbrosset** [comment]
Two quick ideas as I look at this PR:

* Could we make the video links look more like videos? Since they're just links now, it's easy to miss them I guess. We could use the same thumbnail images we use on the https://aka.ms/edge-dev-videos page.
* I'm not too sure about this one, but it felt to me like the videos section should be at the end of the article, not at the beginning. But happy to discuss pros and cons of this.

File: microsoft-edge/devtools-guide-chromium/customize/index.md [hunk 1]
```diff
@@ -113,6 +113,37 @@ To restore the default theme, placement, tools order, and any other customized s
...
+#### Understand the DevTools user interface
+
+_Published on 1 September 2022._
+
+[![Thumbnail image for the Understand the DevTools UI video](./index-images/learn-devtools-ui.png)](https://www.youtube.com/watch?v=ayemJLeE55c)
+
+Understand how the Microsoft Edge DevTools UI is organized. If you are confused about what tools are available, and what the main parts of the DevTools interface do, this video will help get more comfortable.
+
+In this video, we go over the UI structure, with its main toolbar and panel and its drawer area. We also go over how to find, open, and close tools, restore the default settings, and go over the list of available tools.
+
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/customize/index.md:140]: @captainbrosset  - The Welcome text, not included here, includes the word 'customize'.  This section lacks the word 'customize'.  This article's title is "Customize DevTools".  Published URL:
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/

Welcome text:
Check out this video to learn about the main parts of the DevTools UI, how to **customize** it to meet your needs, learn about the different tools that are available, and where to go for help.

**Comment by mikehoffms** [comment]
@captainbrosset - I copied, not linked to, the pngs.  

Now that you've seen the UX with h2 at top of article, this commit instead shows you the h2 at bottom of article - keep at bottom?

File: microsoft-edge/devtools-guide-chromium/customize/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/customize/index.md:140]: When you read this comment thread in main page of PR, you can't see that the lines below it are additional text, from Welcome panel, that include the word 'customize'.

**Comment by captainbrosset** [comment]
> Now that you've seen the UX with h2 at top of article, this commit instead shows you the h2 at bottom of article - keep at bottom?

Yes, I think I prefer keeping the videos at the bottom of the page. The reason for this is the section is quite long. On my monitor it takes the full height of the browser window and, therefore, makes the actual doc content not visible unless I scroll down.

If we had a way to display these videos as a much smaller, and maybe visually different, section then I'd be ok to keep them at the top.

I want to make sure readers can see the content of the documentation right away. To me, the videos are similar to "see also" content.

File: microsoft-edge/devtools-guide-chromium/customize/index.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/customize/index.md:140]: Yes, I think keeping this video in this section makes sense.

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**